modules/deployment/main.tf

```diff
@@ -130,6 +130,13 @@ resource "aws_s3_bucket" "pipeline" {
   tags          = var.tags
 }
 
+resource "aws_s3_bucket_notification" "pipeline" {
```

Comment:
guess we won't need this resource, since we need notifications from the source bucket